ansible/roles/cluster_setup/tasks/sysctl.yml

```diff
@@ -9,5 +9,12 @@
     reload: yes
     sysctl_set: yes
   with_items:
-    - { name: kernel.panic, value: 5 }
+    - name: kernel.hung_task_timeout_secs
+      value: 0
```

Comment:
This is definitely going to make the message go away, since it stops the kernel checking for hung tasks. Without the message, we're not going to know if we fixed the underlying issue.

Reply:
Hmm well we can always put it back if we want to debug.... its just quite annoying as this is part of the reason why nodes won't reboot automatically. By default, we dont want things to hang... this should be an opt in.

Reply:
But this option just controls whether the kernel reports that tasks have hung, it doesn't affect whether tasks will hang.

Reply:
I think this is what is preventing automatic reboot though?

Reply:
It's just for reporting, see https://www.kernel.org/doc/Documentation/sysctl/kernel.txt.

[This article](https://www.nico.schottelius.org/blog/reboot-linux-if-task-blocked-for-more-than-n-seconds/) suggests you can panic on hung tasks by setting hung hung_task_panic=1, from kernel 2.6.35. Then to cause the kernel to reboot on panic, you set kernel.panic=<non-zero number> (as you have done).